benchmark/CMakeLists.txt

```diff
@@ -49,6 +49,19 @@ if(CMAKE_SYSTEM_NAME STREQUAL Darwin)
         LIBRARY_OUTPUT_DIRECTORY "${CMAKE_BINARY_DIR}"
         RUNTIME_OUTPUT_DIRECTORY "${CMAKE_BINARY_DIR}"
     )
+elseif(MSVC)
+    set_source_files_properties(
+        ${PROJECT_SOURCE_DIR}/benchmark/api/query.benchmark.cpp
+        ${PROJECT_SOURCE_DIR}/benchmark/api/render.benchmark.cpp
+        PRIVATE
+            COMPILE_FLAGS "/wd4244"
```

Comment:
Please fix the warnings in non-vendor files.

Reply:
Just to better understand the ability to change files: it's ok to make source code modifications instead supressing warnings? Some places have changes like this piece of code, but I can have the #if _MSVC inside the source code approach as well.

Reply:
I looked up `C4244` which is "floating point truncation". This one can be fixed for all platforms. As such fixes caused regressions in the past, it may be useful to open them in a separate PR.

Reply:
That's nice, I was hoping this change would be welcome. There are are some of these in several files. Using static_cast would make it better for all platforms. I will provide a separate PR to address this.

Reply:
To clarify, I fixed most of those when I did initial Qt port to Windows, but not all files are used there. We only ignore warnings for external packages where we have no direct control.

Reply:
Thanks for the clarification. Now I have a better understanding where I should touch the code, and where I souldn't. My primary concern was to be the least intrusive as I could. But if I have the permission to change some pieces of code (nothing too extensive, just some fixes like this one), I'd rather to.